src/gui/PianoView.cpp

```diff
@@ -618,6 +619,9 @@ void PianoView::mouseMoveEvent( QMouseEvent * _me )
  */
 void PianoView::keyPressEvent( QKeyEvent * _ke )
 {
+#ifdef LMMS_BUILD_LINUX
```

Comment:
But why does left/right matter?  Is the a reason we can't just use both SHIFT keys?